flink-runtime/src/test/java/org/apache/flink/runtime/minicluster/MiniClusterITCase.java

```diff
@@ -400,16 +400,23 @@ public void testJobWithAnOccasionallyFailingSenderVertex() throws Exception {
 		try (final MiniCluster miniCluster = new MiniCluster(cfg)) {
 			miniCluster.start();
 
+			// putting sender and receiver vertex in the same slot sharing group is required
+			// to ensure all senders can be deployed. Otherwise this case can fail if the
+			// expected failing sender is not deployed.
```

Comment:
are they not in the same SSG by default?

Reply:
Yes, by default each JobVertex is in a different slot sharing group, which is aligned to the previous behavior for null slot sharing group before we applies #13321.